modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/query/contributor/ObjectFieldKeywordQueryContributor.java

```diff
@@ -47,6 +47,8 @@ public void contribute(
 			booleanQuery, searchContext, Field.ENTRY_CLASS_PK, false);
 		_queryHelper.addSearchTerm(
 			booleanQuery, searchContext, Field.NAME, false);
+		_queryHelper.addSearchLocalizedTerm(
+			booleanQuery, searchContext, "label", false);
```

Comment:
@marcelabc can you reorder this one too? The `addSearchLocalizedTerm` comes first then the others.